lib/Constants.js

```diff
@@ -59,6 +59,9 @@ module.exports = {
       WINDOW: 'window'
     }
   },
+  priorityCommandClass: [
+    51 // Color Switch
+  ],
```

Comment:
I suggest to move this const inside the function `getPriorityCCFirst` on gateway side for now, and instead of hard coding the 51 use the zwavejs COmmandClasses struct (like we do in the switch case in discover value